march_joint_inertia_controller/src/inertia_estimator.cpp

```diff
@@ -56,9 +55,10 @@ InertiaEstimator::InertiaEstimator(double lambda, size_t acc_size)
   filtered_joint_torque_.resize(fil_tor_size_, 0.0);
 }
 
-double InertiaEstimator::getAcceleration(unsigned int index)
+double InertiaEstimator::getAcceleration()
 {
-  return acceleration_array_[index];
+  auto a = *acceleration_array_.begin();
+  return a;
```

Comment:
Don't go reinventing the wheel :wink: 
```suggestion
  return acceleration_array_.front();
```